lib/juttle-service-client.js

```diff
@@ -0,0 +1,347 @@
+'use strict';
+
+var _ = require('underscore');
+var WebSocket = require('ws');
+var rp = require('request-promise');
+var cli_errors = require('juttle/lib/cli/errors');
+var rp_errors = require('request-promise/errors');
+var sprintf = require('sprintf-js').sprintf;
+
+/*eslint-disable no-console */
+let write_output = console.log;
+let write_error = console.error;
+/*eslint-enable no-console */
+
+let client_exit = process.exit;
+
+// Initialized in handle_command
+let http_url = undefined;
+let ws_url = undefined;
+var API_PREFIX = '/api/v0';
+
+function list_jobs(opts) {
+    let job_id = opts.job || '';
+
+    rp(http_url + '/jobs/' + job_id)
+        .then(function(body) {
+            write_output(JSON.stringify(JSON.parse(body), null, 2));
+        }).error(function(e) {
```

Comment:
I think it was indented that way, just showed up as changed when I moved it out into a standalone function.